src/core/jpx.js

```diff
@@ -18,7 +18,7 @@ import OpenJPEG from "../../external/openjpeg/openjpeg.js";
 
 class JpxError extends BaseException {
   constructor(msg) {
-    super(`JPX error: ${msg}`, "JpxError");
+    super(msg, "JpxError");
```

Comment:
Please don't change this, since it makes JpxError *inconsistent* with the other image-errors (used with JPEG and JBIG2).

Reply:
Ok, I found a bit redundant to have "JpxError" and "JPX error" on the same line.

Reply:
Yes, what I would expect for #17955 is a message that looks like `JPX error: Unknown format`, so basically "JPX error: " plus what the C code emits as specific reason for the failure.

Reply:
Yep I know, but in the console I get:
`Warning: Unable to decode image "img_p0_1": "JpxError: JPX error: Unknown format".`
From my pov, even if change how we dump the warning in the console, it's still redundant... but maybe just a matter of taste.

Reply:
In that case, maybe we should consider changing *all* of `Jbig2Error`, `JpegError`, and `JpxError` to just print `msg` instead?
As long as we're consistent that seems fine to me; @timvandermeij thoughts?

Reply:
If we do that, we'd also have to update all fuzzers because they rely on this prefix in the error message to separate all expected (handled by our code) errors from unexpected (crashes or otherwise unhandled exceptions) errors; please refer to https://github.com/mozilla/pdf.js/blob/master/test/fuzz/jpx_image.fuzz.js#L10-L14. I guess we could catch `JpxError` explicitly there instead, and that'd arguably be nicer than relying on the error _message_ only, but that'd make this a bigger change that I wouldn't recommend doing in this scope (it'd be fine for a follow-up though).